components/Advocates/AdvocatesItemCard.vue

```diff
@@ -42,7 +41,7 @@ const props = withDefaults(defineProps<Props>(), {
 });
 
 // Tags on AppCard is an Array
-const formattedRegion = computed(() => (props.region ? [props.region] : []));
+const formattedRegion = computed(() => (props.region ? props.region : []));
```

Comment:
Now the primary card is not an array, so you don't need to format the region, you can use it directly. Please, remove also the comment